[FIX] 출석 뷰 진입시 앱 종료 오류 수정

## What is this issue? 🛠
출석 뷰 진입시 앱 종료 오류 수정

에러 로그 본문
at org.sopt.official.feature.attendance.AttendanceViewModel
Caused by: java.lang.ClassNotFoundException: Didn't find class "androidx.lifecycle.Transformations"

## Progress 🏃‍♀️
- 출석 뷰 진입시 앱 종료 오류 수정

--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -186,6 +186,7 @@ dependencies {
     implementation(libs.semver)
 
     implementation(libs.code.gson)
+    implementation(libs.lifecycle.livedata.ktx)
 }
 
 secrets {

